File: launchdarkly/ruby/src/PostModelConfigExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["ApiKey"] = "YOUR_API_KEY"
end

model_config_post = LaunchDarklyClient::ModelConfigPost.new
model_config_post.id = "id"
model_config_post.key = "key"
model_config_post.name = "name"
model_config_post.icon = "icon"
model_config_post.provider = "provider"
model_config_post.tags = [
    "tags",
    "tags",
]
model_config_post.params = {}
model_config_post.custom_params = {}

begin
    response = LaunchDarklyClient::AIConfigsBetaApi.new.post_model_config(
        nil, # ld_api_version
        "default", # project_key
        model_config_post,
    )

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling AIConfigsBetaApi#post_model_config: #{e}"
end

File: launchdarkly/python/src/PostModelConfigExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    model_config_post = models.ModelConfigPost(
        id="id",
        key="key",
        name="name",
        icon="icon",
        provider="provider",
        tags=[
            "tags",
            "tags",
        ],
        params={},
        customParams={},
    )

    try:
        response = api.AIConfigsBetaApi(api_client).post_model_config(
            ld_api_version="beta",
            project_key="default",
            model_config_post=model_config_post,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling AIConfigsBetaApi#post_model_config: %s\n" % e)
